.gitignore

```diff
@@ -22,7 +22,9 @@ nuget.exe
 *.ncrunchsolution
 *.*sdf
 *.ipch
-bower_components/
-node_modules/
 .vs/
 project.lock.json
+
+bower_components/
+node_modules/
+**/wwwroot/lib
```

Comment:
I'm removing wwwroot/lib so it will not show up every time npm will download scripts. Ordering change is just for grouping